mobile/appcontainer/FeedbackDialog.ts

```diff
@@ -41,6 +42,9 @@ export const feedbackDialog = hoistCmp.factory({
                 }),
                 button({
                     text: 'Send',
+                    icon: Icon.mail(),
+                    intent: 'primary',
+                    outlined: true,
```

Comment:
Unrelated change - was reviewing and doing a bit of tweaking on our built-in dialogs and their submit/save/apply buttons